scripts/run_metrics.sh

```diff
@@ -38,30 +57,39 @@ if [ ! -f "$FOCAL_OS_RAW_IMAGE" ]; then
 fi
 
 pushd $WORKLOADS_DIR
-grep focal sha1sums-x86_64 | sha1sum --check
+grep focal sha1sums-${TEST_ARCH} | sha1sum --check
 if [ $? -ne 0 ]; then
     echo "sha1sum validation of images failed, remove invalid images to fix the issue."
     exit 1
 fi
 popd
 
 # Build custom kernel based on virtio-pmem and virtio-fs upstream patches
-VMLINUX_IMAGE="$WORKLOADS_DIR/vmlinux"
+if [ ${TEST_ARCH} == "aarch64" ]; then
+       VMLINUX_IMAGE="$WORKLOADS_DIR/Image"
+else
+       VMLINUX_IMAGE="$WORKLOADS_DIR/vmlinux"
+fi
 
 LINUX_CUSTOM_DIR="$WORKLOADS_DIR/linux-custom"
 
 if [ ! -f "$VMLINUX_IMAGE" ]; then
     SRCDIR=$PWD
     pushd $WORKLOADS_DIR
     time git clone --depth 1 "https://github.com/cloud-hypervisor/linux.git" -b "ch-5.15.12" $LINUX_CUSTOM_DIR
-    cp $SRCDIR/resources/linux-config-x86_64 $LINUX_CUSTOM_DIR/.config
+    cp $SRCDIR/resources/linux-config-${TEST_ARCH} $LINUX_CUSTOM_DIR/.config
     popd
 fi
 
 if [ ! -f "$VMLINUX_IMAGE" ]; then
     pushd $LINUX_CUSTOM_DIR
-    time make bzImage -j $(nproc)
-    cp vmlinux $VMLINUX_IMAGE || exit 1
+    if [ $(uname -m) == "x86_64" ]; then
```

Comment:
Nit: You need some consistency with the architecture. Either you use `uname -m` everywhere in the file, or you use `$TEST_ARCH`, but please don't use a mix of the two, as this is a bit confusing.

Reply:
Yeah,